doozerlib/cli/__main__.py

```diff
@@ -2037,16 +2038,27 @@ def get_channel_versions(channel):
 
         return descending_versions, edges
 
+    # New rules, source an OTA managed datsource that specifies minor_floor, minor_block_list,
+    # z_floor, and z_block_list
+    url = f'{suggestions_url}/{major}.{minor}-{arch}.yaml'
+    req = urllib.request.Request(url)
```

Comment:
Do we need a fallback for when this is not available?  Or do we treat that as "no floor" and include all versions?

Reply:
I'm not sure what the right thing to do in absence of data is. I need to test that they can still be overridden and if that's the case then I'd assume we just fall back to a human setting them and filing a ticket with us to get the data for a new release populated. For instance we should have populated 4.6 data by now.

Reply:
From /openshift/cincinnati-graph-data#425, as it stands today:

```yaml
z_floor: 4.6.0
```

Starting off there will block 4.5 -> `4.6.0-[rf]c.*` edges, won't it?  We want those to go into our pre-GA 4.6 releases.

Reply:
Updated that here https://github.com/openshift/cincinnati-graph-data/pull/425/files#diff-8703729926cb5013ad156f1586a9d1c8R7